ffka-gluon-web-usb-wan-hotplug/README.md

```diff
@@ -0,0 +1,9 @@
+# ffka-gluon-web-usb-wan-hotplug
+
+This package adds a new option to the config mode, where you can enable to use
+a USB connected Ethernet adapter as a WAN.
+
+This is especially useful if the tethering function of a mobile phone 
+connected via USB is enabled and to be used as the WAN. For iOS the trust 
+question is automatically triggered, if it is answered positively, iOS also 
+provides an Ethernet device via USB.
```

Comment:
This functionality is not provided by this package, as this one only contains logic to disable / enable ffda-usb-wan-hotplug